Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -494,6 +500,18 @@ i32 Value::to_i32(GlobalObject& global_object) const
     return number.as_i32();
 }
 
+u32 Value::to_u32(GlobalObject& global_object) const
+{
+    auto number = to_number(global_object);
+    if (global_object.vm().exception())
+        return 0;
```

Comment:
```suggestion
        return INVALID;
```

that's just zero but hopefully makes it more clear that the return value of this function is to be discarded in the case of an exception.